repo_files/documentations/bar_chart.md

```diff
@@ -157,7 +157,8 @@ enum values {`start`, `end`, `center`, `spaceEvenly`, `spaceAround`, `spaceBetwe
 ##### Sample 7 ([Source Code](/example/lib/bar_chart/samples/bar_chart_sample7.dart))
 <img src="https://github.com/imaNNeoFighT/fl_chart/raw/master/repo_files/images/bar_chart/bar_chart_sample_7.gif" width="300" >
 
-
+##### Sample 8 ([Source Code](/example/lib/bar_chart/samples/bar_chart_sample8.dart))
+<img src="https://github.com/imaNNeoFighT/fl_chart/raw/master/repo_files/images/bar_chart/bar_chart_sample_8.png" width="300" >
```

Comment:
Can you please take another screenshot from the image?
It's very high resolution and you can't see the details.
<img  width=300 src="https://raw.githubusercontent.com/imaNNeoFighT/fl_chart/c905c3bed69191d69a17b5b59070a0a0f0d23f70/repo_files/images/bar_chart/bar_chart_sample_8.png" />

Reply:
Got a new one pushed